deploy/build.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -e
+set -o nounset
+
+environment=$1
+arch_type=$2
+version=$3
+
+bucket_name=$ED_DEV_SERVERLESS_REPOSITORY_BUCKET
+# if environment is prod then use prod bucket
+if [ "$environment" == "prod" ]; then
+    bucket_name=$ED_SERVERLESS_REPOSITORY_BUCKET
+fi
+
+if [ -z "$arch_type" ]; then
+    echo "Arch type is empty"
+    exit 1
+fi
+
+if [ -z "$version" ]; then
+    echo "Version is empty"
+    exit 1
+fi
+
+project_root=$(git rev-parse --show-toplevel)
+name="edgedelta-extension-layer-${arch_type}"
+ext_path="bin/extensions/$name"
+zip_name="layer_${arch_type}_${version}.zip"
+
+rm -rf "${project_root}/bin"
+mkdir -p "${project_root}/bin/extensions"
```

Comment:
why we first remove bin and then recreate again, just removing under `extensions/` folder isn't enough?